troposphere/firehose.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) 2014, Guillem Anguera <ganguera@gmail.com>
```

Comment:
Is this copyright correct? I'm guessing you copied an existing file but then modified the entire contents? Shouldn't this have 2016 and your name in it?